### PR TITLE
21779 Fixed unit tests

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#21779

*Description of changes:*
- app version = 7.3.4
- clear current panel so other tests re-open it

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/stores/filingHistoryListStore.ts [hunk 1]
```diff
@@ -69,8 +69,7 @@ export const useFilingHistoryListStore = defineStore('filingHistoryList', {
 
     /** The count of filings in the Filing History List. */
     getHistoryCount (): number {
-      const filings = this.getFilings as ApiFilingIF[]
-      return filings.length
+      return this.getFilings.length
```
Comment (severinbeauvais) [inline src/stores/filingHistoryListStore.ts:72]: `this.getFilings` is already typed.

File: tests/unit/FilingHistoryList1.spec.ts [hunk 2]
```diff
@@ -2252,13 +2252,13 @@ describe('Filing History List - with documents', () => {
     submitter: 'Cameron'
   }
 
-  it.skip('does not display the documents list when no documents are present on a filing', async () => {
+  it('does not display the documents list when no documents are present on a filing', async () => {
     // init store
     filingHistoryListStore.setFilings([FILING_WITH_DOCUMENTS_LINK] as any)
+    filingHistoryListStore.setPanel(null) // so we can re-open it
```
Comment (severinbeauvais) [inline tests/unit/FilingHistoryList1.spec.ts:2258]: This is the bug fix.

In this test suite (file), it's the same FHL Store, so unit tests can affect each other. To force the first panel to re-open, this state variable needs to be cleared.

File: tests/unit/FilingHistoryList1.spec.ts [hunk 3]
```diff
@@ -2277,71 +2277,34 @@ describe('Filing History List - with documents', () => {
...
-        ],
-        receipt: 'businesses/CP0000840/filings/112758/documents/receipt'
-      }))
-
-    const wrapper = mount(FilingHistoryList, {
-      propsData: { highlightId: 666 },
-      vuetify
-    })
-    await Vue.nextTick()
-
```
Comment (severinbeauvais) [inline tests/unit/FilingHistoryList1.spec.ts:2344]: I combined this unit test and the one above.

**Review by JazzarKarim — APPROVED** [review]
LGTM

**Review by ArwenQin — APPROVED** [review]
Got it, make sense to me!

**Pull request merged**